taxonomies/additives.txt

```diff
@@ -7710,52 +7849,84 @@ mandatory_additive_class:en: en:acidity-regulator, en:sequestrant, en:stabilizer
+bs:E332(ii), Kalij citrat
 cs:E332(ii), Tripotassium citrate
+cy:E332(ii), Potasiwm sitrad
 da:E332(ii), Trikaliumcitrat
 de:E332(ii), Trikaliumcitrat, Kaliumcitrat
 el:E332(ii), Κιτρικο καλιο
-es:E332(ii), Citrato tripotásico
+eo:E332(ii), Kalia citrato tribaza
+es:E332(ii), Citrato tripotásico, Citrato de potasio
 et:E332(ii), Kaaliumtsitraat
 fi:E332(ii), Trikaliumsitraatti
-fr:E332(ii), Citrate tripotassique
-hu:E332(ii), Trikálium-citrát
+fr:E332(ii), Citrate tripotassique, citrate de potassium
+hu:E332(ii), Trikálium-citrát, Kálium-citrát
 it:E332(ii), Citrato tripotassico
+ja:E332(ii), クエン酸カリウム
 lt:E332(ii), Trikalio citratas
 lv:E332(ii), Trikālija citrāts
 mt:E332(ii), Ċitrat tripotassiku
-nl:E332(ii), Trikaliumcitraat
-pl:E332(ii), Cytrynian tripotasowy
-pt:E332(ii), Citrato tripotássico
+nb:E332(ii), Kaliumsitrat
+nl:E332(ii), Trikaliumcitraat, kaliumcitraat
+pl:E332(ii), Cytrynian tripotasowy, cytrynian potasu
+pt:E332(ii), Citrato tripotássico, citrato de potássio
 ro:E332(ii), Citrat tripotasic
+ru:E332(ii), цитрат калия
+sh:E332(ii), Kalijum citrat
 sk:E332(ii), Citran tridraselný, Kálium-citrát
 sl:E332(ii), Trikalijev citrat
+sr:E332(ii), Kalijum citrat
+sv:E332(ii), Trikaliumcitrat, Kaliumcitrat
 sv:E332(ii), Trikaliumcitrat
+ta:E332(ii), பொட்டாசியம் சிட்ரேட்டு
+te:E332(ii), పొటాషియం సిట్రేట్
+zh:E332(ii), 檸檬酸鉀
+# azb:E332(ii), پوتاسیوم سیترات
+# zh-cn:E332(ii), 柠檬酸钾
+# zh-hans:E332(ii), 柠檬酸钾
+# zh-hant:E332(ii), 檸檬酸鉀
+# zh-hk:E332(ii), 檸檬酸鉀
+# zh-sg:E332(ii), 柠檬酸钾
+# zh-tw:E332(ii), 檸檬酸鉀
+wikidata:en:Q419921
+wikipedia:en:https://en.wikipedia.org/wiki/Potassium_citrate
+# additive/e332ii-tripotassium-citrate has 23 products in 2 languages @2019-04-22
 e_number:en:332
 mandatory_additive_class:en: en:acidity-regulator, en:sequestrant, en:stabilizer
-wikidata:en:Q419921
+wikipedia:en:https://en.wikipedia.org/wiki/Potassium_citrate
+# additive/e332ii-tripotassium-citrate has 23 products in 2 languages @2019-04-22
 additives_classes:en: en:sequestrant, en:stabilizer
 
-en:E333, Calcium citrates , Calcium citrate
+# description:en:CALCIUM CITRATEs is the calcium salt of citric acid. It is commonly used as a food additive (E333), usually as a preservative, but sometimes for flavor. Calcium citrate is also found in some dietary calcium supplements.
+# comment:en:The generic form (E333) is written in plural to distinguish it from E333iii.
+
+en:E333, Calcium citrates
 bg:E333, E333 food additive
 ca:E333, Citrats de calci
 cs:E333, Citrát vápenatý
 da:E333, E333 food additive
-de:E333, Calciumcitrat, Tricalciumcitrat, E 333, Kalziumzitrat, Calciumcitrate
+de:E333, Calciumcitrate
 el:E333, E333 food additive
-es:E333, Citratos de calcio, citrato cálcico, citrato de calcio, Sal amarga, E 333
+es:E333, Citratos de calcio
 et:E333, E333 food additive
 fi:E333, Kalsiumsitraatti
-fr:E333, Citrates de calcium, Citrate de calcium, Citrate de monocalcium, Citrate de dicalcium, Citrate de tricalcium, Citrate tricalcique
```

Comment:
Citrate de monocalcium, Citrate de dicalcium, Citrate de tricalcium should be added to E333(i), (ii) and (iii)

Reply:
See above.